index.html

```diff
@@ -299,17 +213,21 @@ <h1 class="text3">Projects</h1>
         </section>
         <section class="Contact-form" id="contact">
             <img class="img11" src="images/left_topside_icon.svg" alt="Image geometry on the left top side">
-            <form name="Contactform" action="https://formspree.io/f/xbjqawyz" method="POST" aria-required>
+            <form class="mob_form" name="Contactform" action="https://formspree.io/f/xbjqawyz" method="POST" aria-required>
                 <h1 class="text10">Get started</h1>
                 <p class="contact-form-para">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
                 <ul class="contact-form-ullist">
                     <li class="contact-form-name">
                         <input type="text" maxlength="30" class="nooutline" id="name" name="user_name" placeholder="Your name" required>
                         <!--div class="sample_text">Anastasiia</div!-->
                     </li>
-                    <li class="contact-form-email"><input type="email" id="mail" class="nooutline" name="user_email" placeholder="Your email" required></li>
+                    <li class="contact-form-email">
+                        <label for="mail"></label>
+                        <input type="email" id="mail" class="nooutline" name="user_email" placeholder="Your email" required onkeydown="validation()">
+                        <small class="chklabel_email" id="error"></small>
+                    </li>
                     <li class="contact-form-message"><textarea maxlength="500" id="msg" class="nooutline" name="user_message" placeholder="Enter your message here..." required></textarea></li>
-                    <li><button class="contact-form-submit" type="submit">Start collaboration</button></li>
+                    <li><button class="contact-form-submit" id="submitnow" type="submit">Start collaboration</button></li>
```

Comment:
- [ ] when I click on the `submit` button with my email in `Capitalize` format, it doesn't show any error, I get error in the console, please check the screenshot below and fix this
<img width="1679" alt="Screenshot 2021-08-07 at 08 21 41" src="https://user-images.githubusercontent.com/34798898/128592317-357133cd-ddd7-4b38-8bee-8f0efef3c27d.png">

Reply:
Hi. I have corrected the error. Now, no error after clicking the submit button. After clicking on submit button, no error is shown. The problem is fixed. Thank you.